docs/developer-guides/migration-guide-1.4.0.md

```diff
@@ -0,0 +1,114 @@
+# v1.4.0 Migration guide
+
+## Context
+
+This guide describes the process of migrating from Partner Chains SDK v1.3.0 to v1.4.0 for an already
+established chain, avoiding a chain reset.
+
+The biggest change in version v1.4.0 which requires special handling is the update to smart contracts
+version v7.0.0, which:
+- introduces a new governance mechanism which needs to be set up
+- introduces versioned generic datum schema for all datums produced by the smart contracts
```

Comment:
This IMO is an implementation detail. In other words it spends one line of use attention for a small benefit. These datums are invisible for users.